application/src/main/java/org/togetherjava/tjbot/Application.java

```diff
@@ -5,36 +5,48 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.togetherjava.tjbot.commands.CommandHandler;
+import org.togetherjava.tjbot.config.Config;
 import org.togetherjava.tjbot.db.Database;
 
 import javax.security.auth.login.LoginException;
+import java.io.IOException;
 import java.nio.file.Path;
 import java.sql.SQLException;
 
-/***
+/**
  * Main class of the application. Use {@link #main(String[])} to start an instance of it.
  */
 public enum Application {
     ;
 
     private static final Logger logger = LoggerFactory.getLogger(Application.class);
+    private static final String DEFAULT_CONFIG_PATH = "config.json";
 
     /**
      * Starts the application.
      *
-     * @param args command line arguments - [the token of the bot, the path to the database]
+     * @param args command line arguments - [the path to the configuration file (optional, by
+     *        default "config.json")]
      */
     public static void main(final String[] args) {
-        if (args.length != 2) {
-            throw new IllegalArgumentException("Expected two arguments but " + args.length
-                    + " arguments were provided. The first argument must be the token of the bot"
-                    + " and the second the path to the database.");
+        if (args.length > 1) {
```

Comment:
What if `args.length == 0`?

Reply:
Then the code explicitly takes the default value (see the next lines of the code).

Reply:
Ah, yes. Change `arguments were` to `argument(s) was/were` though.

Reply:
Since it is inside the `if (args.length > 1)` it can only trigger for 2 or more arguments. Singular is not an option, hence the text seems to be correct english.